unittests/AST/ASTImporterVisibilityTest.cpp

```diff
@@ -228,12 +246,14 @@ class ImportVisibility
   }
 
   void TypedTest_ImportAfterImportWithMerge() {
-    TranslationUnitDecl *FromTu0 = getTuDecl(getCode0(), Lang_CXX, "input0.cc");
-    TranslationUnitDecl *FromTu1 = getTuDecl(getCode1(), Lang_CXX, "input1.cc");
+    TranslationUnitDecl *FromTu0 =
```

Comment:
Was the original code longer than 80 chars?

Reply:
The "problem" is change from `Lang_CXX` (<80 line length) to `Lang_CXX11` (>80). In a previous PR this was nor fixed at a place.

Reply:
Ok